pkg/cedar/cedarldap/translated_client.go

```diff
@@ -96,6 +97,60 @@ func (c TranslatedClient) FetchUserInfo(ctx context.Context, euaID string) (*mod
 	}, nil
 }
 
+// FetchUserInfos fetches multiple users' personal details
+func (c TranslatedClient) FetchUserInfos(ctx context.Context, euaIDs []string) ([]*models2.UserInfo, error) {
+	if len(euaIDs) == 0 {
+		appcontext.ZLogger(ctx).Error("No EUA ID specified; unable to request user info from CEDAR LDAP")
+		return nil, &apperrors.InvalidParametersError{
+			FunctionName: "cedarldap.FetchUserInfo",
+		}
+	}
+
+	idsStr := strings.Join(euaIDs, ",")
+	params := operations.NewPersonIdsParams()
+	params.Ids = idsStr
+	resp, err := c.client.Operations.PersonIds(params, c.apiAuthHeader)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error(fmt.Sprintf("Failed to fetch person from CEDAR LDAP with error: %v", err), zap.String("euaIDFetched", idsStr))
+		return nil, &apperrors.ExternalAPIError{
+			Err:       err,
+			Model:     models.Person{},
+			ModelID:   idsStr,
+			Operation: apperrors.Fetch,
+			Source:    "CEDAR LDAP",
+		}
+	}
+	if resp.Payload == nil {
+		return nil, &apperrors.ExternalAPIError{
+			Err:       errors.New("failed to return person from CEDAR LDAP"),
+			ModelID:   idsStr,
+			Model:     models.Person{},
+			Operation: apperrors.Fetch,
+			Source:    "CEDAR LDAP",
+		}
+	}
+
+	// If there's nobody returned, we should throw an error
+	if len(resp.Payload.Persons) == 0 || resp.Payload.Persons[0].UserName == "" {
+		return nil, &apperrors.InvalidEUAIDError{
+			EUAID: idsStr,
+		}
+	}
+
+	// Convert the response to our UserInfo model
+	userInfos := []*models2.UserInfo{}
+	for _, person := range resp.Payload.Persons {
+		cedarSys := &models2.UserInfo{
```

Comment:
Variable name is off here, should probably be `userInfo` instead of `cedarSys` for clarity.